opm/parser/eclipse/EclipseState/Schedule/Events.hpp

```diff
@@ -100,7 +100,12 @@ namespace Opm
               GROUP production or injection targets has changed
             */
             GROUP_PRODUCTION_UPDATE = 16384,
-            GROUP_INJECTION_UPDATE = 32768
+            GROUP_INJECTION_UPDATE = 32768,
+
+            /*
+             * New explicit well productivity/injectivity assignment.
+             */
+            WELL_PRODUCTIVITY_INDEX = (1 << 16),
```

Comment:
Using the `(1 << N)` notation is fine - even better than what is there, but then maybe you could update the other events to use the same notation?

Reply:
> maybe you could update the other events to use the [`(1 << N)`] notation?

I've made this change in the latest update.